examples/demo/server/main.go

```diff
@@ -156,6 +158,11 @@ func main() {
 		requestCount.Add(ctx, 1, commonLabels...)
 		span := trace.SpanFromContext(ctx)
 		span.SetAttributes(serverAttribute)
+		bag := baggage.FromContext(ctx)
+		for i, member := range bag.Members() {
+			span.AddEvent(fmt.Sprint("baggage member: ", i), trace.WithAttributes(attribute.String(member.Key(), member.Value())))
```

Comment:
is this code reading baggage, and adding them as spanevents? Why not simply add them as span attributes?

Reply:
make sense, I will update it.